### PR TITLE
PP-238 Add basic instrumentation for newrelic

### PR DESCRIPTION
**Pull request by zadsas**
- Include -javaagent flag for the JVM
- Add newrelic folder
- Remove *.jar from .gitignore, as newrelic*-.jar is needed

### CODE REVIEW

**Comment by ijmad** [comment]
:+1: except could you undo the .gitignore change and instead add the newrelic JAR with a git add -f? We don't want other JAR files accidentally polluting our repos without good reason.

File: newrelic/newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+# This section is for settings common to all environments.
+# Do not add anything above this next line.
+common: &default_settings
+
+  # ============================== LICENSE KEY ===============================
+  # You must specify the license key associated with your New Relic
+  # account. For example, if your license key is 12345 use this:
+  # license_key: '12345'
+  # The key binds your Agent's data to your account in the New Relic service.
+  license_key: 'b7b28104622602d27e2119ea754a9b8e10a81fb8'
```
Comment (ijmad) [inline newrelic/newrelic.yml]: Repeating Dan's comment from https://github.com/alphagov/pay-publicauth/pull/11, this needs to be an environment variable.

**Comment by surfraz** [comment]
Closing for now, Going to re-open after PR for pay-ansible to support using Env variables to provide application name and key.

**Comment by ijmad** [comment]
+1

**Pull request closed without merging**